app/src/main/java/com/sunasterisk/coinqapp/ui/coindetail/info/CoinInfoFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.sunasterisk.coinqapp.ui.coindetail.info
+
+import androidx.core.os.bundleOf
+import com.sunasterisk.coinqapp.base.BaseFragment
+import com.sunasterisk.coinqapp.databinding.FragmentCoinInfoBinding
+
+class CoinInfoFragment : BaseFragment<FragmentCoinInfoBinding>() {
+
+    override val binding by lazy { FragmentCoinInfoBinding.inflate(layoutInflater) }
+
+    override fun initViews() {
+    }
+
+    override fun initListeners() {
+    }
+
+    companion object {
+        private const val BUNDLE_COIN_ID = "BUNDLE_COIN_ID"
+
+        fun getInstance(coinId: String) = CoinInfoFragment().apply {
```

Comment:
```suggestion
        fun newInstance(coinId: String) = CoinInfoFragment().apply {
```
tương tự nơi khác, tránh nhầm lần với SIngleton chỗ khác của e

Reply:
h em sửa thì pull bị to lên ạ. A châm trước giúp em với